integration-cli/docker_cli_top_test.go

```diff
@@ -8,28 +8,57 @@ import (
 )
 
 func (s *DockerSuite) TestTopMultipleArgs(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	out, _ := dockerCmd(c, "run", "-i", "-d", "busybox", "top")
+	out, _ := runSleepingContainer(c, "-d")
 	cleanedContainerID := strings.TrimSpace(out)
 
-	out, _ = dockerCmd(c, "top", cleanedContainerID, "-o", "pid")
-	c.Assert(out, checker.Contains, "PID", check.Commentf("did not see PID after top -o pid: %s", out))
+	if daemonPlatform != "windows" {
+		out, _ = dockerCmd(c, "top", cleanedContainerID, "-o", "pid")
+		c.Assert(out, checker.Contains, "PID", check.Commentf("did not see PID after top -o pid: %s", out))
+	} else {
+		// Windows does not support arguments to `docker top`
+		_, _, err := dockerCmdWithError("top", cleanedContainerID, "-o", "pid")
+		c.Assert(err.Error(), checker.Contains, "Windows does not support arguments to top", check.Commentf("expected Windows to fail top with arguments", err.Error()))
+
```

Comment:
@dnephin :+1: Neat, didn't know about iCmd. Updated as per your suggestion, but its `Out` rather than `Stdout`, and `ExitCode` needed setting for the failure case :innocent: